Android.mk

```diff
@@ -67,11 +67,17 @@ LOCAL_CFLAGS = \
     -D__STDC_CONSTANT_MACROS \
     -D__STDC_LIMIT_MACROS \
     -D__UMD \
-	-Wno-error
+    -Wno-error \
+    -msse4 \
```

Comment:
Hi @arun1778 , 
Currently, the sse related compile flag missed in Celadon BoardConfig. We need to add this to our project. It's not a good idea. Could we add it to the Celadon BoardConfig, so every application can get this benefit?

thanks